stdlib/testing/chronograf/aggregate_window_count_test.flux

```diff
@@ -68,7 +69,6 @@ inData = "
 ,,1,2018-05-22T00:00:40Z,mem,percentage,host.remote,35
 ,,1,2018-05-22T00:00:50Z,mem,percentage,host.remote,35
 "
-
```

Comment:
Probably fine with this, though we often add this whitespace for reasons. I _kinda_ wish the formatter just left it alone, but I don't want to scope creep this patch. Maybe an issue is warranted?